code/modules/holiday/holidays.dm

```diff
@@ -420,11 +384,80 @@
 				begin_day += 31
 				begin_month-- //begins in march, ends in april
 
-//	to_chat(world, "Easter calculates to be on [begin_day] of [begin_month] ([days_early] early) to [end_day] of [end_month] ([days_extra] extra) for 20[yy]")
 	return ..()
 
 /datum/holiday/easter/celebrate()
 	..()
 	GLOB.maintenance_loot += list(
 		/obj/item/weapon/reagent_containers/food/snacks/egg/loaded = 15,
 		/obj/item/weapon/storage/bag/easterbasket = 15)
+
+/datum/holiday/easter/greet()
+	return "Greetings! in the far flung past of [current_year], today would be easter, Be sure to have a happy one and keep an eye out for Bunnies!"
+
+
+/datum/holiday/easter/spess
+	name = SPACE_EASTER
+	year_offset = 540 //Canonicially the year is CURRENT YEAR + 540, so this is Easter as it would be IN SS13 itself
+
+/datum/holiday/easter/spess/greet()
+	return "Greetings! the year is [current_year+year_offset], and it's easter! be sure to have a happy one and keep an eye out for Easter Bunnies!"
```

Comment:
Easter needs to always be capitalized and bunnies does not have to be capitalized